dockers/docker-swss-layer-bullseye/Dockerfile.j2

```diff
@@ -0,0 +1,22 @@
+{% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
+FROM docker-config-engine-bullseye
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+
+{% if docker_swss_layer_bullseye_debs.strip() -%}
```

Comment:
>docker_swss_layer_bullseye_debs [](http://example.com/codeflow?start=6&length=31)

The "-layer-" name is special. Could you just call it "docker-swss-buster"?

Reply:
I am keeping convention with the existing `docker_swss_layer_buster`. As I understand it, multiple dockers require the same set of packages which is why we have the `docker_swss_layer_buster`. Once all the dockers are upgraded to bullseye, would the same be true, and we would require this `docker_swss_layer_bullseye` for common packages?